internal/ir/xds.go

```diff
@@ -207,6 +207,8 @@ type HTTPListener struct {
 	// HTTP3 provides HTTP/3 configuration on the listener.
 	// +optional
 	HTTP3 *HTTP3Settings `json:"http3,omitempty"`
+	// Path contains settings for path URI manipulations
+	Path PathSettings `json:"path,omitempty"`
```

Comment:
ptr type

Reply:
It doesn't make sense to have a pointer here. This is not an optional structure.

Reply:
oops, my bad, not noticing